src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -111,6 +113,43 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
     addWelcomeMessage()
   }
 
+  @RequiresEdt
+  private fun addSubscriptionTab() {
+    val activeAccountType = CodyAuthenticationManager.instance.getActiveAccount(project)
+    if (activeAccountType != null && activeAccountType.isDotcomAccount()) {
+      tryRestartingAgentIfNotRunning(project)
+      val server = getServer(project)
+      if (server != null) {
+        val codyProFeatureFlag = server.evaluateFeatureFlag(GetFeatureFlag("CodyProJetBrains"))
+        if (codyProFeatureFlag.get() != null &&
+            codyProFeatureFlag.get()!! &&
+            java.lang.Boolean.getBoolean("cody.isGa")) {
```

Comment:
As we decided during the meetings. Let's drop `cody.isGa` condition here. We will request ff exceptions for ones that want to test these features.